handler/actions/src/main/java/io/knotx/fragments/handler/action/http/options/HttpActionOptions.java

```diff
@@ -53,7 +53,7 @@ public String getHttpMethod() {
    * @param httpMethod HTTP method
```

Comment:
> At the moment only HTTP GET method is supported.

That's not true any more :)
Pease mention the default method, if none specified.